src/inject/fallback.ts

```diff
@@ -0,0 +1,8 @@
+const fallbackStyle = document.createElement('style');
+fallbackStyle.textContent = 'html,body,body :not(iframe) { background-color:#181a1b!important;border-color:#776e62!important;color:#e8e6e3!important;';
+document.documentElement.appendChild(fallbackStyle);
```

Comment:
Not sure if it works in FF.

Reply:
Firefox is already know to prevent whiteflashes also have great customizations likes shadowFox what even prevent it more. It's rather chromium and it's flashes as

> Even on a high-end system I noticed a nice white flash, however with this I didn't notice it. 

This was on ungoogled chromium and stable google chrome. And the high end system is fast fast and renders things within seconds like gmail(with dark reader enabled on dynamic mode).

Reply:
Oh this for sure works to prevent #3523 On Firefox(Low-end system is very noticeable on firefox for lightmode, now It's not with this change for sure)